source/escalonador.c

```diff
@@ -56,6 +208,20 @@ bool receive_message( message_t *message_received, int queue_id )
 Auxiliary methods for the delay_execution module ------------------------------------------------------
 */
 
+static void alarm_handler(int signo)
+{
+   printf("Caught a signal to delay\n");
```

Comment:
Actually the signal is to push into execution the head of the execution_queue.

Reply:
isso ai ja estava na realidade hahah mas eu realmente nao me toquei. Mas ele ta funcionando dessa forma sim.

Reply:
Olha o cara quebrando a vibe do Inglês aí. hhahaha.
Beleza, a gente muda no próximo commit.